estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=fields.Date.today()+relativedelta(months=3), copy=False)
```

Comment:
Just a small note:
`fields.Date.today()` will actually give you the server time, if you want to have the actual today of the user (which is the case most of the time for business logic) you'll have to use `context_today`.